content/04-guides/01-database-workflows/08-sql-views.mdx

```diff
@@ -0,0 +1,378 @@
+   }
+
+   model User {
+     email String  @unique
+     id    Int     @default(autoincrement()) @id
+     name  String?
+     Post  Post[]
+   }
+
+   + model Draft {
+   +   title     String
+   +   id        Int     @unique
+   +   email     String
+   +   published Boolean
+   + }
+   ```
+
+## 6. Generate Prisma Client
+
+In this section, you will generate Prisma Client.
+
+1. Add a `generator` block to your Prisma schema (typically added right below the `datasource` block):
+
+   ```prisma
+   generator client {
+     provider = "prisma-client-js"
+   }
+   ```
+
+2. Run the following command to install and generate Prisma Client in your project:
+
+   ```
+   npx prisma generate
+   ```
+
+Now you can use Prisma Client to send database queries in Node.js.
+
+## 9. Validate the Drafts view in a Node.js script
+
+In the following section, you will use the `drafts` model property to return `Post` records that have not yet been published. To use the `drafts` model property:
+
+1. Create a new file named `index.js` and add the following code to it:
+
+   ```ts
+   const { PrismaClient } = require('@prisma/client')
+
+   const prisma = new PrismaClient({})
+
+   async function main() {
+     const sarahPosts = await prisma.user.create({
+       data: {
+         name: 'Sarah',
+         email: 'sarah@prisma.io',
+         Post: {
+           create: [
+             { title: 'My first post', published: false },
+             { title: 'All about databases', published: true },
+             { title: 'Prisma Day 2020', published: false },
+           ],
+         },
+       },
+     })
+
+     const emilyPosts = await prisma.user.create({
+       data: {
+         name: 'Emily',
+         email: 'emily@prisma.io',
+         Post: {
+           create: [
+             { title: 'My first post', published: false },
+             { title: 'All about databases', published: true },
+             { title: 'Prisma Day 2020', published: false },
+           ],
+         },
+       },
+     })
+
+     const drafts = await prisma.drafts.findMany({})
+
+     console.log(drafts)
+
+     const filteredDrafts = await prisma.drafts.findMany({
+       where: {
+         email: 'sarah@prisma.io',
+       },
+     })
+
+     console.log(filteredDrafts)
+   }
+
+   main()
+     .catch(e => {
+       throw e
+     })
+     .finally(async () => {
+       await prisma.disconnect()
+     })
+   ```
+
+   This example:
+
+   - Creates two `User` records with three `Post` records each
+   - Returns all `Draft` records from the view
+   - Returns all `Draft` records from the view where the author's email is `emily@prisma.io`
+
+2. Run the code with the following command:
+
+   ```
+   node index.js
+   ```
+
+   The following output indicates that the view works as expected - the first query returns all drafts (`Post` records where `published` is `false`), and the second query returns drafts by `emily@prisma.io` only):
+
+   ```json
```

Comment:
this needs to be `js` instead of `json` (because the keys are not using double quotes)

Reply:
ah actually, because of the extra outputs `ALL DRAFTS` and `FILTERED DRAFTS` this isn't valid `js` either, but maybe it still highlights